PowerValidatedSolutions.psm1

```diff
@@ -32398,18 +32405,47 @@ Function Test-EsxiAuthentication {
 Export-ModuleMember -Function Test-EsxiAuthentication
 
 Function Test-VsphereConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to a vCenter Server instance.
+
+        .DESCRIPTION
+        Checks the network connectivity to a vCenter Server instance.
+        Supports testing a connection on ports 443 (HTTPS) and 22 (SSH). Default: 443 (HTTPS).
+
+        .EXAMPLE
+        Test-VsphereConnection -server sfo-m01-vcf01.sfo.rainpole.io
+        This example checks network connectivity with a vCenter Server instance on default port, 443 (HTTPS).
+
+        .EXAMPLE
+        Test-VsphereConnection -server sfo-m01-vcf01.sfo.rainpole.io -port 443
+        This example checks network connectivity with a vCenter Server instance on port 443 (HTTPS). This is the default port.
+
+        .EXAMPLE
+        Test-VsphereConnection -server sfo-m01-vcf01.sfo.rainpole.io -port 22
```

Comment:
```suggestion
        Test-VsphereConnection -server sfo-m01-vc01.sfo.rainpole.io -port 22
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.